client/src/components/Button.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  text-align: center;
+  width: 280px;
+  height: 66px;
+  background: #f85c23;
+  color: #0e0f09;
+  border: 3px #f85c23 solid;
+  border-radius: 30px;
+  box-shadow: 0px 3px 6px grey;
+  font-size: 26px;
```

Comment:
I would recommend setting` font-size` in `px` at the root level of your app, which is the `html` element. That way you can use `rem` here.
Check out [this](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97) article